### ISSUE TITLE
Support volumes mounted multiple times as read-only

### ISSUE THREAD

**Issue by robszumski** [coreos/torus#234]
It'd be great to support the ability to mount a single volume into multiple Kubernetes pods as read-only.

Possible use-cases:

 - web root shared between a bunch of nginx instances
 - seed or cache some data for an application

**Comment by rimusz** [comment]
+1

**Comment by robszumski** [comment]
@rimusz Can you expand on what your use-case is for this?

**Comment by rimusz** [comment]
there are many kind of statefull Apps which need to share the same files:
e.g. Wordpress setup with Management Pod to make updates and the rest of Pods just read the shared files as read only

**Comment by alekssaul** [comment]
I believe JIRA enterprise in cluster setup also requires this.

**Comment by barakmich** [comment]
Yeah, let's do it. This turns the virtual Mutex in `block/{metadata,etcd}.go` into a RWMutex.

**Comment by aleksijohansson** [comment]
It seems this is not yet implemented even though v0.1.2 is out? Would be super important feature also in web applications like Drupal although the ultimate goal would be this https://github.com/coreos/torus/issues/235
